### PR TITLE
post: cli v8.1

### PR DESCRIPTION
**Pull request by cexbrayat**
- [x] remove TODOs
- [x] update post date

### CODE REVIEW

File: _posts/2019-07-03-angular-cli-8.1.md [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+    # all specs in src/app/admin/*
+    ng test --include app/admin
+    # just src/app/user.service.spec.ts
+    ng test --include app/user.service.spec.ts
+
+You could already focus a test suite by changing the `describe`
+function to `fdescribe` or the `it` function to  `fit`,
+but `ng test` was still rebuilding all your tests.
+By specifying `--include`, only the provided files are rebuilt,
+so it's faster and doesn't care if you have compilation errors in other files 🌈.
```
Comment (jnizet) [inline _posts/2019-07-03-angular-cli-8.1.md]: And I would add:

And of course, you won't forget to replace `fdescribe` by `describe` before committing anymore.

File: _posts/2019-07-03-angular-cli-8.1.md [hunk 2]
```diff
@@ -0,0 +1,176 @@
...
+to allow updating when the repository contains modified or untracked files.
+Previously, trying to run `ng upgrade` in a project with unsaved modifications
+was always resulting in an error.
+
+## AoT by default for Ivy
+
+The framework team has been working hard at squashing bugs in Ivy,
+and also improved the build mechanisms,
+resulting in faster AoT builds.
+The CLI team now considers that the performances with AoT and Ivy enabled
```
Comment (jnizet) [inline _posts/2019-07-03-angular-cli-8.1.md]: Haven't you made a change in the book saying that it's enabled by default, even without Ivy?

File: _posts/2019-07-03-angular-cli-8.1.md [hunk 3]
```diff
@@ -0,0 +1,176 @@
...
+but then the AoT rebuilds are roughly as fast as the JiT rebuilds without Ivy✨!
+
+## TypeScript configuration changes
+
+A side-note about the TypeScript configuration to conclude.
+If you check out the
+[differences between version 8.0 and 8.1](https://github.com/cexbrayat/angular-cli-diff/compare/8.0.0...8.1.0),
+you'll notice that the `tsconfig.json` file lost the `emitDecoratorMetadata` option.
+This option was only needed for JiT development
+for the dependency injection (you can [read on old blog post of mine about that](/2016/12/08/angular-injectable)).
```
Comment (jnizet) [inline _posts/2019-07-03-angular-cli-8.1.md]: on old -> an old

File: _posts/2019-07-03-angular-cli-8.1.md (hunk 2 above)
Comment (cexbrayat) [reply _posts/2019-07-03-angular-cli-8.1.md]: No it's in an unmerged PR for when lvy lands https://github.com/Ninja-Squad/angular-ebook/pull/606

Comment (jnizet) [reply _posts/2019-07-03-angular-cli-8.1.md]: Ah OK. Thanks. So it will be merged in 5 months, when Angular 9 gets outs :-)

Comment (cexbrayat) [reply _posts/2019-07-03-angular-cli-8.1.md]: Exactly :) I first thought that it was for every app, before realizing it was only for Ivy apps. But as I had already written it, I kept the PR...

**Pull request merged**